bigtable/client/client_options.cc

```diff
@@ -28,5 +28,55 @@ ClientOptions::ClientOptions() {
   channel_arguments_ = grpc::ChannelArguments();
 }
 
+void SetCompressionAlgorithm(grpc::ChannelArguments *channel_arguments,
```

Comment:
Any reason these functions are not inlined?

Reply:
Done.